services/src/main/java/org/keycloak/services/resources/admin/RoleContainerResource.java

```diff
@@ -172,7 +172,6 @@ public void deleteRole(final @PathParam("role-name") String roleName) {
             throw new NotFoundException("Could not find client");
         }
 
-        RoleRepresentation rep = getRole(roleName);
```

Comment:
was not used ... and didn't seem to have a relevant side effect so I removed it.